src/emulator/gxm/src/gxm.cpp

```diff
@@ -820,7 +745,7 @@ namespace texture {
         case SCE_GXM_TEXTURE_ADDR_CLAMP:
             return GL_CLAMP_TO_EDGE;
         case SCE_GXM_TEXTURE_ADDR_MIRROR_CLAMP:
-            return GL_MIRROR_CLAMP_TO_EDGE;
+            return GL_CLAMP_TO_EDGE; // FIXME: Not supported in OpenGL 4.1 core.
```

Comment:
Why not supported? My NVIDIA card before that PR used to use OGL 4.6 (according to log) and this work fine. Doc too doesn't tell anything regarding it being unsupported.

Reply:
I read something about `GL_CLAMP` being removed in 3.1, but not `GL_CLAMP_TO_EDGE` yeah.

Reply:
`CLAMP_TO_EDGE` is supported, which is why I used it, but `GL_MIRROR_CLAMP_TO_EDGE` is available only in OGL 4.4+. I'll update the comment to be clearer, sorry.